Cargo.toml

```diff
@@ -12,6 +12,11 @@ exclude = ["/link_tests"]
 rust-version = "1.49" # 1.48 doesn't support `ManuallyDrop` in union.
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[dependencies]
+abi_stable = { version = "0.10", default-features = false, optional = true }
 
 [dev-dependencies]
 tokio = { version = "1.4.0", features = ["macros", "rt-multi-thread", "sync", "time"] }
+
+[features]
+sabi = ["abi_stable"]
```

Comment:
```suggestion
```

I'd just use the dependency name `abi_stable` as the feature name.

Reply:
Unfortunately, as far as I know, having a feature with the same name as a dependency is impossible with Cargo.

Reply:
> Unfortunately, as far as I know, having a feature with the same name as a dependency is impossible with Cargo.

Why? Just removing this line and using `#[cfg(feature = "abi_stable")]` should be enough. An optional dependency automatically creates a feature with the same name.

Reply:
Oh what? I didn't know that! Will try to use that then, thanks.

Reply:
```suggestion
```

You forget to delete this.

Reply:
Sorry, done!